blazor/data/how-to/change-update-type-from-PATCH-to-PUT.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: change the update type from PATCH to PUT in Blazor DataManager Component | Syncfusion
+description: Checkout and learn here all about changing the update type from PATCH to PUT in Syncfusion Blazor DataManager component and more.
+platform: Blazor
+control: DataManager
+documentation: ug
+---
+
+<!-- markdownlint-disable MD024 -->
+
+# Change the update type from PATCH to PUT
+
+By using [SfDataManager](https://help.syncfusion.com/cr/aspnetcore-blazor/Syncfusion.Blazor.Data.SfDataManager.html) instance, we can change the default UpdateType request from "PATCH" to "PUT.
```

Comment:
explain ODataV4 adaptor support PATCH update type and its purpose. then why we are switching between these two update types